### PR TITLE
release v1.6.0

### PR DESCRIPTION
**Pull request by incertia**
let me know if i missed anything

### CODE REVIEW

**Comment by arcz** [comment]
We should emphasize that slither is now required

**Comment by ggrieco-tob** [comment]
> We should emphasize that slither is now required

That's why I think this should be 1.6.0

**Review by ggrieco-tob — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## TODO
 
-## 1.5.2
+## 1.6.0
+* Slither is now mandatory to properly run Echidna.
```
Comment (incertia) [inline CHANGELOG.md:4]: slither is now a required dependency

Comment (ggrieco-tob) [reply CHANGELOG.md]: Fixed.

**Review by ggrieco-tob — APPROVED** [review]

**Pull request merged**